__tests__/app.test.js

```diff
@@ -34,17 +34,17 @@ describe('GET /api/topics', () => {
             response.body.topics.forEach((topic) => {
             expect(typeof topic.description).toBe('string');
             expect(typeof topic.slug).toBe('string');
-            expect.objectContaining({
+            expect(response.body.topics[0]).toEqual(expect.objectContaining({
                 description: expect.any(String),
                 slug: expect.any(String)
-              })
+              }))
```

Comment:
Really close with this - lines 35 & 36 can be removed now that you are using object containing syntax. Instead of `response.body.topics[0]` here that tests only one object you can use `topic` as you are still inside a forEach.